### DISCUSSION TITLE
Allow showing desktop notifications only for the Noisy level with keeping unread counts for group chats

### DISCUSSION THREAD

**Discussion by MurzNN** [element-hq/element-meta#1823] [Product Feedback]
#### What would you like to do?
I want to see desktop notifications for new messages in only one-to-one chats and mentions, but not for group chats!

#### Why would you like to do it?
I have a lot of group chats, and periodically read them, so I want to keep seeing the unread messages count on each group chat. But I don't want to be disturbed by any new message in them.
And this is a default behavior for Telegram, WhatsApp, and most other instant messengers, so I wanna have the same in Element.

#### How would you like to achieve it?
We have three options for each notification type: 
- Noisy
- On
- Off

With "Off" mode I will not see the count of unread messages in rooms, so this is not a solution for this case.

So the solution seems to be to create a new option "Enable desktop notifications only for Noisy" that will cover this feature.

**Comment by t3chguy** [comment]
The popup notification experience should be consistent between clients

**Comment by daniellekirkwood** [comment]
Moving this to discussion as it's not something in our backlog right now. We've just started to land product updates to notifications on Element Web and Desktop. Please check them out and provide feedback back here! Thanks :) 

(for now they can be found in Labs)
